src/styles/Blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 16px;
```

Comment:
it should be `32px` padding from the top 😉
don't combine padding and margins in children, better just set certain padding